indra/llcommon/llsys.cpp

```diff
@@ -454,14 +454,14 @@ LLOSInfo::LLOSInfo() :
 
 #ifndef LL_WINDOWS
 // static
-S32 LLOSInfo::getMaxOpenFiles()
+long LLOSInfo::getMaxOpenFiles()
```

Comment:
You prefer `long` here and not `S64` or `int64_t`?

Reply:
I changed to `long` because the compiler scolded me for trying to return specifically `long` (the return type of the system function) through `S32`. I was intentionally matching the type.